MIG-1531: MTC docs missing Direct Migration Requirements (#73269)
openshift/openshift-docs: migration_toolkit_for_containers/installing-mtc.adoc

* Run an Rsync pod as root on the destination cluster per migration.

> In both cases, you must set the following labels on the source side of any namespaces that are running workloads with higher privileges prior to migration: `enforce`, `audit`, and `warn.`

Author: ```suggestion
In both cases, you must set the following labels on the source side of any namespaces that are running workloads with higher privileges before the migration: `enforce`, `audit`, and `warn.`
```
